regexes/device/mobiles.yml

```diff
@@ -5570,10 +5854,16 @@ LG:
       model: 'Optimus Bright L-07C'
     - regex: '(?:LG-)?F800(?:S|K|L)?'
       model: 'V20'
-    - regex: 'LM-Q710(?:\(FGN\)|\.FGN)'
+    - regex: 'LM-Q710(?:\(FGN\)|\.FGN|FGN)'
       model: 'Q Stylus Plus'
-    - regex: 'LM-Q710\.FG'
+    - regex: 'LM-Q710\.(FG)'
```

Comment:
Any reason for adding brackets here? There seems to be no use of the matching result

Reply:
A more precise definition of the device

```
    - regex: 'LM-Q710(?:\(FGN\)|\.FGN|FGN)'
      model: 'Q Stylus Plus'

   - regex: 'LM-Q710\.(FG)'
      model: 'Q7'
```
Merge?

What to do with other versions "Q8, Q8+, Q9

Reply:
My point was simply that `LM-Q710\.FG` and `LM-Q710\.(FG)` expresses the same if `$1` is not used in the model name...

Reply:
fix